### PR TITLE
Add warning regarding user added adaptation-common

### PR DESCRIPTION
**Pull request by rainemak**
Add warning regarding user added adaptation-common. Users should
never add adaptation-common themselves rather it should be the one
coming from the global ssu settings.

See JB#57866

### CODE REVIEW

File: Reference/Sailfish_OS_Cheat_Sheet/README.md [hunk 1]
```diff
@@ -404,8 +404,22 @@ ssu lr                  # list
 ssu updaterepos
 pkcon refresh
 pkcon update
-```    
-    
+```
+
+Never override *adaptation-common* repository as that will messup your device when upgrading it. Before upgrade make sure *adaptation-common* repository is not listed in *Enabled repositories (user):* as follows:
```
Comment (saukko) [inline Reference/Sailfish_OS_Cheat_Sheet/README.md:409]: This is not only adaptation-common but any global repository, right?

Comment (sledges) [inline Reference/Sailfish_OS_Cheat_Sheet/README.md:409]: nit:
```suggestion
Never override *adaptation-common* repository as that will mess up your device when upgrading it. Before upgrade make sure *adaptation-common* repository is not listed in *Enabled repositories (user):* as follows:
```

Comment (rainemak) [reply Reference/Sailfish_OS_Cheat_Sheet/README.md:409]: Thanks.

Comment (rainemak) [reply Reference/Sailfish_OS_Cheat_Sheet/README.md]: I guess it's better to rephrase that way just to be on the safe side.

Comment (rainemak) [reply Reference/Sailfish_OS_Cheat_Sheet/README.md]: Adjusted

**Review by mkosola — APPROVED** [review]

**Pull request merged**